AutoRest/Generators/Ruby/Azure.Ruby/Templates/AzureMethodTemplate.cshtml

```diff
@@ -1,19 +1,18 @@
 @using System
-@using Microsoft.Rest.Generator.ClientModel
 @using Microsoft.Rest.Generator.Ruby
 @using Microsoft.Rest.Generator.Ruby.TemplateModels
 @using Microsoft.Rest.Generator.Ruby.Templates
 @inherits Microsoft.Rest.Generator.Template<Microsoft.Rest.Generator.Azure.Ruby.AzureMethodTemplateModel>
 
-@if (!Model.IsLongRunningOperation)
+@if (!Model.IsLongRunningOperation && !Model.IsPageable)
 {
 @:@(Include(new MethodTemplate(), (MethodTemplateModel)Model))
 }
-else if (Model.HttpMethod == HttpMethod.Post || Model.HttpMethod == HttpMethod.Delete)

```

Comment:
curious how come we don't need differenciator between (Post || Delete) and other? Did we change something or it was dead code?